samples/msgext-ai-sentiment-analysis/nodejs/.env

```diff
@@ -0,0 +1,6 @@
+MicrosoftAppId=<<MicrosoftAppId>>
+MicrosoftAppPassword=<<MicrosoftAppPassword>>
+BaseUrl=<<BOT_ENDPOINT>>
+SECRET_OPENAI_API_KEY=<<SECRET_OPENAI_API_KEY>>
```

Comment:
Are you using it?
If not please remove it.

Reply:
Yes using, and I have tested locally with this values